eq-author/src/apollo/createApolloErrorLink.js

```diff
@@ -9,14 +9,19 @@ import { signOutUser } from "../redux/auth/actions";
 
 export const errorHandler = (getStore, error) => {
   const { networkError, graphQLErrors } = error;
+
   setSentryUser(window.localStorage.getItem("accessToken"));
-  if (networkError) {
+  if (networkError && networkError.bodyText === "User does not exist") {
+    getStore().dispatch(signOutUser());
+  } else if (
```

Comment:
Can this be refactored so it doesn't have multiple nested if statements?

Reply:
Using a switch statement to check for status code.